topology/parameter.h

```diff
@@ -897,7 +897,7 @@ class DifferenceParameter : public Parameter
   }
 
 protected:
-  Parameter *parameter1_, *parameter2_;
+  Parameter* parameter1_, *parameter2_;
 };
 
 /**
```

Comment:
This file has nothing to do with the pull request and just got in due to clang reformatting. Please remove from PR.